pkg/kubelet/pod_workers.go

```diff
@@ -375,9 +374,12 @@ type podWorkers struct {
 	// Tracks by UID the termination status of a pod - syncing, terminating,
 	// terminated, and evicted.
 	podSyncStatuses map[types.UID]*podSyncStatus
-	// Tracks when a static pod is being killed and is removed when the
-	// static pod transitions to the killed state.
-	terminatingStaticPodFullnames map[string]struct{}
+	// Tracks all full names for known pods
+	podFullnames map[types.UID]string
```

Comment:
It would be better to put the full name in the `podSyncStatus` struct vs having another map to maintain here (unless podFullnames has a different set than podSyncStatuses, which would be confusing and likely to cause issues).